binary-transcoders/src/main/kotlin/com/kamelia/sprinkler/transcoder/binary/encoder/composer/ComposedEncoder.kt

```diff
@@ -0,0 +1,78 @@
+@file:JvmName("ComposedEncoderFactory")
+
+package com.kamelia.sprinkler.transcoder.binary.encoder.composer
+
+import com.kamelia.sprinkler.transcoder.binary.encoder.UTF8StringEncoder
+import com.kamelia.sprinkler.transcoder.binary.encoder.core.Encoder
+import com.kamelia.sprinkler.transcoder.binary.encoder.core.EncoderOutput
+import java.nio.ByteOrder
+
+/**
+ * Creates a new encoder of type [T] using the given lambda [block]. The [block] parameter is a lambda accepting an
```

Comment:
Not sure about using the term "lambda" here to talk about functions in general

```suggestion
 * Creates a new encoder of type [T] using a given function. The [block] parameter is a function accepting an
```